app/api/__init__.py

```diff
@@ -10,6 +11,13 @@
 # users
 api.route(UserList, 'user_list', '/users')
 api.route(UserDetail, 'user_detail', '/users/<int:id>')
```

Comment:
add an endpoint `/nofications/<int:notification_id>/user` similar to how '/computers/<int:computer_id>/owner' is implemented on http://flask-rest-jsonapi.readthedocs.io/en/latest/quickstart.html#first-example. Do the same for the events api